### PR TITLE
[CHORE] Bump version to 0.2.0

### PR DESCRIPTION
**Pull request by manh-t**
## What happened 👀

Bump version to 0.2.0

## Insight 📝

N/A

## Proof Of Work 📹

N/A

### CODE REVIEW

**Comment by manh-t** [comment]
🚀 Deployed on https://64a2a34257067b72e093f067--manh-react-survey.netlify.app

**Comment by manh-t** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Congrats, coverage is good</td>
    </tr>
  </tbody>
</table>

## Coverage in All Files
File | Line Coverage | Statement Coverage | Function Coverage | Branch Coverage
---- | ------------: | -----------------: | ----------------: | --------------:
[src/App.tsx](../blob/016e5b0/src/App.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
[src/index.tsx](../blob/016e5b0/src/index.tsx) | (3/3) 100% | (3/3) 100% | (0/0) NaN% | (0/0) NaN%
[src/../index.tsx](../blob/016e5b0/src/components/ElevatedButton/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
[src/../TextInput/index.tsx](../blob/016e5b0/src/components/TextInput/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (2/2) 100%
[src/lib/requestManager.ts](../blob/016e5b0/src/lib/requestManager.ts) | (5/5) 100% | (5/5) 100% | (2/2) 100% | (1/1) 100%
[src/routes/index.tsx](../blob/016e5b0/src/routes/index.tsx) | (1/1) 100% | (1/1) 100% | (0/0) NaN% | (0/0) NaN%
[src/screens/SignIn/index.tsx](../blob/016e5b0/src/screens/SignIn/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
Total | (21/21) 100% | (21/21) 100% | (6/6) 100% | (3/3) 100%

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0f1c88180e56a8370157d0e3be3713adc5e832d5
</p>

File: public/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="root-container"></div>
```
Comment (nvminhtue) [inline public/index.html]: normally, we don't style for this div, it's purely just used to inject the React App to this DOM. Let's add the style on the App Root component

File: src/assets/stylesheets/layouts/main/index.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+html,
+body {
+  top: 0;
```
Comment (nvminhtue) [inline src/assets/stylesheets/layouts/main/index.scss]: I guess you wanted to reset the CSS? If you did, let's use [this](https://meyerweb.com/eric/tools/css/reset/) for standardizing approach.
Unless please skip this comment but don't forget to adjust regarding the above comment

**Comment by manh-t** [comment]
@nvminhtue that files appeared due to the error when rebasing. Fixed 😄

**Review by nvminhtue — APPROVED** [review]

**Review by tyrro — APPROVED** [review]

**Pull request merged**